.github/workflows/linux.yml

```diff
@@ -100,9 +106,28 @@ jobs:
         with:
           name: ${{ env.LAYOUT }}-release-binary
           path: target.tar.gz
+      # Filters that automagically detect if WPT test for specific layout needs to be run
+      - uses: dorny/paths-filter@v2
+        if: ${{ inputs.wpt == 'auto' }}
+        id: filter
+        with:
+          filters: |
+            layout2020:
+              - 'components/layout/**'
+              - 'components/layout_2020/**'
+              - 'components/layout_thread_2020/**'
+              - '**-layout-2020/**'
+            layout2013:
+              - 'components/layout/**'
+              - 'components/layout_2013/**'
+              - 'components/layout_thread_2013/**'
```

Comment:
This filter wildcards is the most crucial part of the PR.

Reply:
Hrm. This is a bit suspicious because there is no `layout_2013` and `layout_thread_2013` directories. The `layout` and `layout_thread` directories correspond to Layout 2013.

Reply:
Good catch.